server/athenian/api/models/web/team.py

```diff
@@ -0,0 +1,86 @@
+# coding: utf-8
+
+from datetime import date, datetime
+
+from typing import List, Dict, Type
+
+from athenian.api.models.base_model_ import Model
+from athenian.api.models.user import User
+from athenian.api import util
+
+
+class Team(Model):
+    """NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+
+    Do not edit the class manually.
+    """
+
+    def __init__(self, name: str = None, members: List[User] = None):
+        """Team - a model defined in OpenAPI
+
+        :param name: The name of this Team.
+        :param members: The members of this Team.
+        """
+        self.openapi_types = {"name": str, "members": List[User]}
+
+        self.attribute_map = {"name": "name", "members": "members"}
+
+        self._name = name
+        self._members = members
+
+    @classmethod
+    def from_dict(cls, dikt: dict) -> "Team":
+        """Returns the dict as a model
+
+        :param dikt: A dict.
+        :return: The Team of this Team.
+        """
+        return util.deserialize_model(dikt, cls)
+
+    @property
+    def name(self):
+        """Gets the name of this Team.
+
+        Name of the team.
+
+        :return: The name of this Team.
+        :rtype: str
+        """
+        return self._name
+
+    @name.setter
+    def name(self, name):
```

Comment:
```suggestion
    def name(self, name: str):
```
... and so on. The default generated code is not ideal.